.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ executors:
     working_directory: ~/app
   builder:
     docker:
-      - image: circleci/node:12.16.1-buster-browsers
+      - image: circleci/node:12-buster-browsers
```

Comment:
possibly controversial, but this doesn't tie the node version to a specific point release.
(so will actually upgrade to 12.18)

Reply:
(this was a bit of an accidental change but thought it might be worth leaving it in.)